.github/workflows/releases.yml

```diff
@@ -80,11 +84,24 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ github.token }}
 
-      - name: Login to Dockerhub (release)
-        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
It would be good if you can change this to SHA reference as tags can be modified on source repo to point different commit

Reply:
updated